package.json

```diff
@@ -51,18 +54,32 @@
     "react-test-renderer": "18.0.0",
     "semantic-release": "^19.0.3"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run pretty-quick"
-    }
-  },
```

Comment:
This can still be kept, to make sure prettier runs automatically with each commit.

This way people won't have the "surprise" of wrongly formatted files once they open a PR

Reply:
WIth `husky` 1.x that was indeed done automatically as husky installed itself on `postinstall` hook. This is not true anymore with Husky newer versions, as users need to opt-in to use husky be running `husky install`. https://blog.typicode.com/husky-git-hooks-autoinstall/

Re formatting files: I think that IDE format on save is now primary way to format the code.

Reply:
Having a `prepare` script that installs `husky` will still make that it's installed automatically with newer versions. 🤷‍♂️

IDEs can have this disabled, so it's always a good idea to enforce it for sure imo.

Reply:
I've reverted Husky removal, I will handle Husky upgrade in a separate PR.